api-documentation/invoice.md

```diff
@@ -649,10 +658,11 @@ Partner API Key
 
 {% endapi-method-response-example-description %}
 
-```text
+```javascript
 {
   "import_id": "...", 
   "external_invoice_number": "2014-342-545", 
+  "reference": "internal-reference",
```

Comment:
I would replace this with something random. "internal-reference" is ambiguous, can be internal for CC? or internal for the partner?